core/Config.php

```diff
@@ -157,7 +157,8 @@ public function getClientSideOptions()
             'action_url_category_delimiter' => $general['action_url_category_delimiter'],
             'autocomplete_min_sites' => $general['autocomplete_min_sites'],
             'datatable_export_range_as_day' => $general['datatable_export_range_as_day'],
-            'datatable_row_limits' => $this->getDatatableRowLimits()
+            'datatable_row_limits' => $this->getDatatableRowLimits(),
+            'are_ads_enabled' => $general['piwik_pro_ads_enabled']
```

Comment:
I named it `are_ads_enabled` instead of `piwik_pro_ads_enabled` as I wasn't sure if WhiteLabel plugin would replace the word Piwik there and lead to problems in the end